src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -665,9 +643,7 @@
 
 		<!-- domain.c: mono_defaults.thread_class -->
 		<!-- FIXME: -->
```

Comment:
@marek-safar @akoeplinger @vargaz - do you know what this `FIXME` is here for? Are we fixing it with this change?

Reply:
@vargaz any recollection why this was added?